.hooks/chech_mdx_changes.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
File name is wrong.

Reply:
And do we need this script?
I think we can just always run `make build-doc` and `make gen-doc` directly in the hooks.

Reply:
[tested](https://github.com/instill-ai/component/pull/251/files)

Reply:
can we merge this because it is not user-face PR.